MusicNotationKit/MusicNotationKit/Types.swift

```diff
@@ -70,7 +70,8 @@ public enum Dot {
 
 public enum Accent {
 	case Standard
-	case Strong
+    case Strong
```

Comment:
Something is a little odd with the tabbing. You should have your Xcode set to 4 tabs and be using tabs instead of spaces. Make sure those settings are the same so it matches the rest of the file.